src/pages/studyView/studyPageHeader/ActionButtons.tsx

```diff
@@ -68,8 +68,15 @@ export default class ActionButtons extends React.Component<
                 }
             );
 
-            // if number of cases large equal to MAXIMUM_NAV_CASE_IDS_IN_URL, use clientPostedData to pass navCaseIds
-            if (navCaseIds.length >= MAXIMUM_NAV_CASE_IDS_IN_URL) {
+            const url = getPatientViewUrl(
+                firstPatient.studyId,
+                firstPatient.patientId,
+                navCaseIds
+            );
+
+            // if length is great than 4500 it will crash browsers
+            // so use alternative method passing ids
+            if (url.length > 4500) {
```

Comment:
should we also define a constant for this? something like:
```JavaScript
const MAX_ALLOWED_URL_LENGTH = 4500;
```

Reply:
done